components/raftstore/src/store/peer.rs

```diff
@@ -5161,6 +5161,15 @@ where
             send_msg.set_end_key(region.get_end_key().to_vec());
         }
 
+        match msg.get_msg_type() {
+            MessageType::MsgAppend | MessageType::MsgHeartbeat => {
```

Comment:
Need to consider hibernate region as well. If in hibernate state and apply index is updated, send an extra message explicitly.

Reply:
Fixed, thanks!